workflow/snakemake_rules/main_workflow.smk

```diff
@@ -1304,11 +1304,13 @@ rule assign_rbd_levels:
         tree = "results/{build_name}/tree.nwk",
     params:
         config=config["files"]["rbd_level_definitions"],
-        basal_clade_label="21L (Omicron)"
+        basal_clade_label="21L (BA.2)"
```

Comment:
Typo in the message of d7bb79a56d30c20afb341c4ddd86421123036c71?

```diff
- implies that this shouldn't not be configurable
+ implies that this shouldn't be configurable
```

Reply:
Whoops! Thanks for catching that, fixed in 90a84cdd1e9f11fb4ecbe5184eb284a85d0ec29d